File: problems/rubyshousehunt/attachments/template.cpp
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int income;
    float mr;
    int savings;
    cin >> income >> mr >> savings; cin.ignore();
    int n;
    cin >> n; cin.ignore();
    for (int i = 0; i < n; i++) {
        int price;
        int expenses;
        cin >> price >> expenses; cin.ignore();
    }

    // Write an answer using cout. DON'T FORGET THE "<< endl"
    // To debug: cerr << "Debug messages..." << endl;

    cout << "Yes, the house is affordable with 1 roommate(s) and $1 to spare." << endl;
}

File: problems/rubyshousehunt/attachments/template.py
import sys
import math

inputs = input().split()
income = int(inputs[0])
mr = float(inputs[1])
savings = int(inputs[2])
n = int(input())
for i in range(n):
    price, expenses = [int(j) for j in input().split()]

# Write an answer using print
# To debug: print("Debug messages...", file=sys.stderr, flush=True)

print("Yes, the house is affordable with 1 roommate(s) and $1 to spare.")
